lmdeploy/turbomind/chat.py

```diff
@@ -66,9 +67,13 @@ def main(model_path: str,
             else:
                 new_kwargs[k] = v
         kwargs = new_kwargs
+
+    engine_cfg = TurbomindEngineConfig(**kwargs)
```

Comment:
not all kwargs are for engine_config. You should filter kwargs

Reply:
Thanks for the suggestion. Actually, after the filter for `ChatTemplateConfig`, all the keys left in `kwargs` are `TurbomindEngineConfig`'s attributes. Previously, @AllentDan also mentioned that. But anyway, adding the filter logic seems safer.

Reply:
And also I forgot to pass `model_name` and `tp`. Just fixed.

Reply:
Great.